scripts/us_bea/states_gdp/README.md

```diff
@@ -1,22 +1,90 @@
-# Importing US Bureau of Economic Analysis quarterly GDP data per US state
+# Importing US Bureau of Economic Analysis quarterly GDP data per US state into Data Commons
 
-## Raw Data
-- Available for download at [https://www.bea.gov/data/gdp/gdp-state](https://www.bea.gov/data/gdp/gdp-state).
+Author: fpernice-google
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Procedure](#Import-Procedure)
+
+## About the Dataset
+
+### Download URL
+
+ZIP file is available for download from [https://www.bea.gov/data/gdp/gdp-state](https://www.bea.gov/data/gdp/gdp-state).
+
+### Overview
+
+> Gross Domestic Product (GDP) measures the overall level of economic activity in a country or region. The US Department of Commerce’s Bureau of Economic Analysis (BEA) publishes various forms of economic data, including GDP, consumer spending, income and saving, prices, inflation, and employment. Specifically on GDP, it publishes data at the national, state, industry and county levels, among others. In this import, we focus on overall US state GDP data, as well as per industry per US state GDP.
+
+> Data Format: All considered data is measured in either 2012 chained US Dollars or current US Dollars. The former means that the data is inflation-adjusted to 2012 money, by using the inflation rate measured by the chain-weighted CPI (i.e. taking into account product substitutions). The latter simply means nominal GDP. In both cases, during the cleaning process, this data gets converted from millions of USD to USD (i.e. by multiplying by one million).
+
+This dataset is broken up into 2 major families of variables:
+1. US state GDP: Quarterly GDP per US state.
+2. US state industry GDP: Quarterly GDP per industry in each US State.
+
+US state GDP is further broken down into:
+1. Chained 2012 USD: GDP measured in chained 2012 USD.
+2. Nominal USD: GDP measured in nominal (current) USD.
+
+US state industry GDP is further broken down into:
+1. Industry code 11: Agriculture, forestry, fishing and hunting.
```

Comment:
Sigh.  This seems overly prescribed.  Let's leave it though.  No changes suggested.